apps/frontend/src/components/global/Sidebar.vue

```diff
@@ -1,65 +1,419 @@
 <template>
-  <v-navigation-drawer
-    :value="value"
-    :clipped="$vuetify.breakpoint.lgAndUp"
-    app
-    :style="{'margin-top': classification ? '5em' : '3.5em', 'z-index': 11}"
-    disable-resize-watcher
-    disable-route-watcher
-    fixed
-    temporary
-    width="600px"
-    @input="$emit('input', $event)"
-    @blur="value = false"
-  >
-    <v-expansion-panels v-model="active_path" accordion>
-      <DropdownContent
-        header-text="Results"
-        :files="visible_evaluation_files"
-        :all-selected="all_evaluations_selected"
-        :enable-compare-view="true"
-        :compare-view-active="compareViewActive"
-        @toggle-all="toggle_all_evaluations"
-        @toggle-compare-view="compareView"
-        @changed-files="$emit('changed-files')"
-      />
-      <DropdownContent
-        header-text="Profiles"
-        :files="visible_profile_files"
-        :all-selected="all_profiles_selected"
-        @toggle-all="toggle_all_profiles"
-        @changed-files="$emit('changed-files')"
-      />
-    </v-expansion-panels>
-  </v-navigation-drawer>
+  <div>
+    <ChecklistTargetDataModal
+      :visible="showTargetModal"
+      style="display: none"
+      @close-modal="showTargetModal = false"
+    />
+    <ChecklistTechnologyAreaModal
+      :visible="showTechnologyModal"
+      style="display: none"
+      @close-modal="showTechnologyModal = false"
+    />
+
+    <v-navigation-drawer
+      :clipped="$vuetify.breakpoint.lgAndUp"
+      app
+      :style="{'z-index': 11}"
+      disable-resize-watcher
+      disable-route-watcher
+      fixed
+      permanent
+      width="45px"
+      @input="$emit('input', $event)"
+      @blur="value = false"
+    >
+      <v-container v-if="!isSideUtilityDrawerShown" fill-height fluid>
+        <v-row align="center" justify="center">
+          <v-col
+            ><v-icon
+              class="text-center"
+              right
+              @click="isSideUtilityDrawerShown = !isSideUtilityDrawerShown"
+              >mdi-arrow-right</v-icon
+            ></v-col
+          >
+        </v-row>
+      </v-container>
+    </v-navigation-drawer>
+    <v-navigation-drawer
+      v-model="isSideUtilityDrawerShown"
+      :clipped="$vuetify.breakpoint.lgAndUp"
+      app
+      :style="{'z-index': 11, 'margin-top': classification ? '5em' : '3em'}"
```

Comment:
set up a full heimdall server deployment on your machine and test out what this looks like when the classification banner is just like the one word cui/fouo or something obnoxiously long.  i think we're probably fine with the assumption that it'll only be one line long, but worthwhile asking eugene/aaron if they expect it to ever be multiple lines long.  if that case happens, then we're gonna need to figure out how to get the dynamic height and adjust accordingly.